index.html

```diff
@@ -322,9 +350,64 @@ <h3>FLOSS at Geek Feminism</h3>
 				<p><a href="#">♥</a></p>
 			</div>
 		</footer>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
We already load jQuery further down, or do you need the latest version for that? (Or is the .slim version not enought?)

Reply:
It seems that the ajax calls dont work with the other jquery script tag, need to look into this.